cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -437,18 +437,18 @@ && rawRebalanceAnnotation(kafkaRebalance) == null) {
                                                 kafkaRebalance.getMetadata().getName(), desiredStatusAndMap.getLoadMap())
                                                 .compose(i -> updateStatus(reconciliation, currentKafkaRebalance, desiredStatusAndMap.getStatus(), null))
                                                 .compose(updatedKafkaRebalance -> {
-                                                    String message = "";
+                                                    String message;
```

Comment:
Why does the `message` obejct need to be defined outside of the `if` scope? It seems to be used only there.

Reply:
Agree. It was this way before because we had two different `if` at the same level using `message` but now it can be moved inside.

Reply:
Done